docs/reference/commandline/network_inspect.md

```diff
@@ -201,6 +201,97 @@ $ docker network inspect ingress
 ]
 ```
 
+### Using `verbose` option for `network inspect`
+
+`docker network inspect --verbose` for swarm mode overlay networks shows service-specific details like VIP and port mappings. It also
```

Comment:
How about this ?
`docker network inspect --verbose for swarm mode overlay networks shows service-specific details such as service's VIP and port mappings.`